### ISSUE TITLE
Updating jackson version to 2.15

### ISSUE THREAD

**Issue by jan-tosovsky-cz** [networknt/json-schema-validator#793]
jackson 2.14 branch depends on the vulnerable snakeyaml 1.33 version, while the 2.15 branch (released recently) depends on the 2.0 version. 
https://github.com/FasterXML/jackson-dataformats-text/blob/2.14/yaml/pom.xml
```
<dependency>
    <groupId>org.yaml</groupId>
    <artifactId>snakeyaml</artifactId>
    <version>1.33</version>
</dependency>
```

**Comment by stevehu** [comment]
I have executed a light-bot script to upgrade all the repositories to Jackson 2.15.1. I am doing the build and test at the moment to see if there are any broken changes. So far, there is only one issue with the timezone, and it is minor. If we have any issues, we can roll back easily.

**Comment by stevehu** [comment]
We have an issue with JDK 11 build with one of the plugins introduced recently. I have opened a ticket on the plugin moditect repo. 

https://github.com/moditect/moditect/issues/200

**Comment by jan-tosovsky-cz** [comment]
That was quick! Thanks a lot. 

Btw, this CVE was reported by the Dependency Checker tool. Sometimes it is not clear if this could be somehow exploited. When I saw a fixed version, I simply filed this issue.

**Comment by mr-zepol** [comment]
Just out of curiosity, when this change will be released? I know version 1.0.83 was released last week but wondering about when a new release will happen, thanks

**Comment by stevehu** [comment]
We are doing more tests to ensure that there is no issues with this version of Jackson. Once our test is completed, we will release another version. Thanks.
